webui/package.json

```diff
@@ -30,36 +31,27 @@
     "validator": "^13.7.0"
   },
   "devDependencies": {
+    "@types/bootstrap": "^4.6.2",
+    "@types/jest": "^29.0.3",
+    "@types/lodash": "^4.14.185",
+    "@types/react": "^17.0.50",
+    "@types/react-dom": "^17.0.17",
+    "@types/react-router-dom": "^5.3.3",
+    "@types/react-syntax-highlighter": "^15.5.5",
+    "@types/react-test-renderer": "^17.0.2",
+    "@types/validator": "^13.7.7",
+    "@typescript-eslint/eslint-plugin": "^5.38.0",
+    "@typescript-eslint/parser": "^5.38.0",
     "@vitejs/plugin-react-refresh": "^1.3.1",
     "babel-eslint": "^10.1.0",
     "eslint-plugin-react": "^7.23.2",
-    "jest": "^27.3.1",
+    "jest": "^29.0.3",
     "react-test-renderer": "^17.0.2",
+    "ts-jest": "^29.0.2",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.8.3",
+    "typesync": "^0.9.2",
     "vite": "^2.3.7",
```

Comment:
consider moving to vide 3

Reply:
Thanks for the feedback. I'm not sure what (if any) the breaking changes are. I'll look into it.